batcher/aligned/src/main.rs

```diff
@@ -392,14 +392,22 @@ async fn main() -> Result<(), AlignedError> {
                     }
                     Err(e) => {
                         warn!("Error while submitting proof: {:?}", e);
-                        handle_submit_err(e).await;
-                        return Ok(());
+                        handle_submit_err(&e).await;
+                        // In the case of an InsufficientBalance error we record and continue processing the entire msg queue.
```

Comment:
what do we record?
and why do we want to continue processing the message queue? maybe there is a replacement message in the submit_multiple? or is there another case?